Git-commit: 98f836e80d210312ec98ffbbce4a0b6c4b6dc34e
From: Helge Deller <deller@gmx.de>
Date: Thu, 27 Oct 2022 09:12:05 +0200
Subject: [PATCH] parisc: Export iosapic_serial_irq() symbol for serial port
 driver
References: bsn#19
Patch-mainline: v5.10.154

commit a0c9f1f2e53b8eb2ae43987a30e547ba56b4fa18 upstream.

The parisc serial port driver needs this symbol when it's compiled
as module.

Signed-off-by: Helge Deller <deller@gmx.de>
Reported-by: kernel test robot <lkp@intel.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/parisc/iosapic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/parisc/iosapic.c b/drivers/parisc/iosapic.c
index 8a3b0c3a1e92..fd99735dca3e 100644
--- a/drivers/parisc/iosapic.c
+++ b/drivers/parisc/iosapic.c
@@ -875,6 +875,7 @@ int iosapic_serial_irq(struct parisc_device *dev)
 
 	return vi->txn_irq;
 }
+EXPORT_SYMBOL(iosapic_serial_irq);
 #endif
 
 
-- 
2.35.3

